src/WaterPokemon.java

```diff
@@ -0,0 +1,88 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class WaterPokemon extends Pokemon{
+    private final List<String> attacks;
+
+    public WaterPokemon(String name, int level, int healthPoints, String food, String speech) {
+        super(name, level, healthPoints,"water",speech, food);
+        this.attacks = setAttacks();
+    }
+
+    private List<String> setAttacks() {
```

Comment:
Heel goed dat je deze private hebt gemaakt. Het is een handige helper methode om je constructor schoon en leesbaar te houden.